odoo/addons/base/res/res_config.py

```diff
@@ -545,9 +548,11 @@ def execute(self):
                 implied_group.write({'users': [(3, user.id) for user in groups.mapped('users')]})
 
         # other fields: execute all methods that start with 'set_'
+        # and the method 'set_fields'
         for method in dir(self):
```

Comment:
Like with method `get_default_fields`, this `for` loop may be deprecated in favor of the single method `set_fields`...
(You may log a warning if setter methods are actually called this way.)